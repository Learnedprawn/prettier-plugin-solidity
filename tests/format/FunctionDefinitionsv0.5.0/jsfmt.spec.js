run_spec(import.meta, ['solidity-parse']);
run_spec(import.meta, ['solidity-slang-parse'], {compiler: '0.5.0'});
