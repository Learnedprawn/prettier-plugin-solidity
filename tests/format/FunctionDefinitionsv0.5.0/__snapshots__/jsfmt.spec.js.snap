// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultFunctionDefinitions.sol - {"compiler":"0.5.0"} format 1`] = `
====================================options=====================================
compiler: "0.5.0"
parsers: ["solidity-slang-parse"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
contract FunctionDefinitions {
  function () external {}
  function () external payable {}
}

=====================================output=====================================
contract FunctionDefinitions {
    function() external {}
    function() external payable {}
}

================================================================================
`;

exports[`DefaultFunctionDefinitions.sol format 1`] = `
====================================options=====================================
parsers: ["solidity-parse"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
contract FunctionDefinitions {
  function () external {}
  function () external payable {}
}

=====================================output=====================================
contract FunctionDefinitions {
    function() external {}
    function() external payable {}
}

================================================================================
`;
